ui/components/Footer.tsx

```diff
@@ -16,7 +17,34 @@ const RightFoot = styled(Flex)`
 
 const LeftFoot = styled(Flex)``;
 
+const REPO_URL = "https://github.com/weaveworks/weave-gitops";
+
 function Footer({ className }: Props) {
+  const { data, isLoading } = useVersion();
+  const versionData = data?.version || {} as Version;
+  const apiVersion: Version = {
+    version: versionData.version,
+    gitCommit: versionData["git-commit"],
+    branch: versionData.branch,
+    buildTime: versionData["buildtime"]
+  };
+
+  const shouldDisplayApiVersion = !isLoading
+    && apiVersion.version !== p.version
+    && apiVersion.branch
+    && apiVersion.gitCommit;
+
+  const versionText = shouldDisplayApiVersion
+    ?
+    `${apiVersion.branch}-${apiVersion.gitCommit}`
```

Comment:
The formatting looks a little weird here. Are you using `prettier` and formatting on save? If not, we will get weird diffs when the next person saves this file.

Reply:
Added running prettier on save and prettified the files changed.